velox/docs/configs.rst

```diff
@@ -636,6 +636,13 @@ Each query can override the config by setting corresponding query session proper
        Standard mode is built on top of legacy mode and has throttled retry enabled for throttling errors apart from transient errors.
        Adaptive retry mode dynamically limits the rate of AWS requests to maximize success rate.
 
+Bucket Level Configuration
+""""""""""""""""""""""""""
+All hive.s3 config (except hive.s3.log-level) can be set on a per-bucket basis. The bucket-specific option is set by
```

Comment:
Should we highlight a bit better when the string is part of the config? Maybe use bold or italic? This might be easier to read?

Reply:
I enclosed in double quotes to be consistent with highlights at other places.